src/main/java/com/atlassian/db/replica/api/AuroraMultiReplicaConsistency.java

```diff
@@ -10,12 +10,13 @@
     private final ReplicaConsistency replicaConsistency;
     private final AuroraClusterDiscovery cluster;
 
-    private AuroraMultiReplicaConsistency(
-        ReplicaConsistency replicaConsistency,
-        AuroraConnectionDetails auroraConnectionDetails
-    ) {
+    public AuroraMultiReplicaConsistency(ReplicaConsistency replicaConsistency, AuroraClusterDiscovery cluster) {
```

Comment:
`AuroraClusterDiscovery` is internal and leaks to the API (you need an instance of AuroraClusterDiscovery to use AuroraMultiReplicaConsistency).

Reply:
removed the leak